services/logging-operator/4.2.5/extra-images.txt

```diff
@@ -1,4 +1,4 @@
 {{ .Values.fluentbit.image.repository }}:{{ .Values.fluentbit.image.tag }}
```

Comment:
We need to bump the service version from `4.2.5` to `4.2.6` and all references in the service file.